src/bun.js/api/bun/spawn.zig

```diff
@@ -22,7 +22,7 @@ fn _getSystem() type {
 const Environment = bun.Environment;
 const system = _getSystem();
 
-const Maybe = JSC.Node.Maybe;
```

Comment:
By using `JSC.Maybe` we can continue to associate that one with `sys.Error` and allow JSC.Node.Maybe to be `ErrorType` agnostic. Doing this is the least invasive way to allow more configurability in the `Maybe`.